port/win/env_win.cc

```diff
@@ -7,6 +7,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file. See the AUTHORS file for names of contributors.
 
+#if defined(OS_WIN)
```

Comment:
I personally prefer putting compilation conditions inside the source file as much as possible, to simplify build scripts. If others disagree, speak up!